packages/components/button/src/use-button.ts

```diff
@@ -223,6 +223,7 @@ export function useButton(props: UseButtonProps) {
     spinnerSize,
     disableRipple,
     getButtonProps,
+    onClear,
```

Comment:
Something like this would be better:
```suggestion
    getRippelProps: () => ({ rippels, onClear }),
```

Reply:
@jguddas Hello, I have completed the modification, please review again, thank you